Mathlib/Data/Finite/Set.lean

```diff
@@ -29,11 +29,10 @@ variable {α : Type u} {β : Type v} {ι : Sort w}
 
 theorem Finite.Set.finite_of_finite_image (s : Set α) {f : α → β} (h : s.InjOn f)
     [Finite (f '' s)] : Finite s :=
-  Finite.of_equiv _ (Equiv.ofBijective _ h.bij_on_image.Bijective).symm
+  Finite.of_equiv _ (Equiv.ofBijective _ h.bijOn_image.bijective).symm
 #align finite.set.finite_of_finite_image Finite.Set.finite_of_finite_image
 
 theorem Finite.of_injective_finite_range {f : ι → α} (hf : Function.Injective f)
     [Finite (range f)] : Finite ι :=
   Finite.of_injective (Set.rangeFactorization f) (hf.codRestrict _)
```

Comment:
It turns out we need `ι : Sort _` in `Data.Set.Function` (broken in https://github.com/leanprover-community/mathlib4/pull/1035/commits/7df512689919df7efa7821f782212bf9b5d3ac39 @mcdoll )